### PR TITLE
Muon analysis raw plot speed

### LINKED ISSUE #32379: Slower loading of runs in Muon Analysis and Frequency Domain Analysis
**Original reporter:** Peter Baker ISIS

**Describe the bug**
During Beta testing it was taking longer to load runs in the Muon Analysis and Frequency Domain Analysis GUIs than in version 6.1. This is the initial loading of the run and creates a single plot. It is also noticeable that changing Instrument on the Home Tab is slower too in comparison to v.6.1. The stack trace doesn't seem to suggest that loading is taking any longer but it definitely feels like it is.

**To Reproduce**
Open Mantid v 6.1
Open Muon Analysis or Frequency Domain Analysis
Load MUSR 62260 or EMU 85022

Compare with 
Opening Mantid Nightly
Open Muon Analysis or Frequency Domain Analysis
Load MUSR 62260 or EMU 85022

**Expected behavior**
Runs load as fast as in previous versions

**Platform/Version (please complete the following information):**
 - OS: Windows 10
 - Mantid Version: 6.1.20210831.1435

**Additional context**
Changes to Load were made recently as part of #32021 which might be linked to this issue?

### PR DESCRIPTION
**Pull request by AnthonyLim23**
**Description of work.**

The loading speed of muon analysis was slower than previous version as it was creating 16 tiled plots.

So we have disabled the raw plotting.

**To test:**

Open muon analysis
Load some data
The load speed should be faster than without this PR and a normal work flow should be fine

Do the same for the frequency domain analysis GUI and a normal work flow should be fine

Fixes #32379

*This does not require release notes* because it fixes a speed issue of a new feature.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/Muon/GUI/Common/plot_widget/raw_pane/raw_pane_presenter.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def __init__(self, view, model, context,figure_presenter):
     def handle_new_data(self, autoscale=True, hold_on=False):
         self._model.check_num_detectors()
         self.update_selectors()
-        self.handle_data_updated(autoscale,hold_on)
+        #self.handle_data_updated(autoscale,hold_on)
```
Comment (AnthonyLim23) [inline scripts/Muon/GUI/Common/plot_widget/raw_pane/raw_pane_presenter.py:32]:
```suggestion
```

**Review by Harrietbrown — DISMISSED** [review]
With this PR I estimate an approximate 20% (3-4 seconds) speed up in loading. This is great although the load time is still noticeable. A good start for speeding up the interface.

**Comment by AnthonyLim23** [comment]
Im going to push some more changes as David has found some related speed ups

**Comment by DavidFair** [comment]
> Im going to push some more changes as David has found some related speed ups

I'm going to mark this as a draft and dismiss the review then until the work is done

Also could we test the functionality beyond load times, especially around plotting as we may have unintentional breakage from this too

**Comment by AnthonyLim23** [comment]
after discussion with users, the decision is to disable it from a user's point of view

**Comment by AnthonyLim23** [comment]
@Harrietbrown could you please have another look?

**Review by Harrietbrown — CHANGES_REQUESTED** [review]
This is a massive improvement, loading is up to 4 times faster with these changes. but can you remove the commented lines? unless they are being kept there to be used in a latter PR?

**Comment by AnthonyLim23** [comment]
Its commented out so we can add it back in easily after the release

**Review by Harrietbrown — APPROVED** [review]
This massively speeds up the loading in Muon analysis and FDA. Good job.

**Comment by AnthonyLim23** [comment]
@mantidproject/gatekeepers could we get this merged please? It is blocking another piece of work

**Pull request merged**